src/rust/ensogl/lib/core/src/gui/cursor.rs

```diff
@@ -251,8 +251,11 @@ crate::define_endpoints! {
     }
 
     Output {
-        position        (Vector3),
-        screen_position (Vector3),
+        position              (Vector3),
+        screen_position       (Vector3),
+        scene_position        (Vector3),
+         /// Change between the current and the previous screen position.
```

Comment:
the frp name is about scene and docs are about screen. I believe the docs are wrong.